INSTRUCTION.md

```diff
@@ -0,0 +1,19 @@
+# ToDo App Instructions
+
+## Docker Hub Repository
+The Docker image for this application is available at:
+[https://hub.docker.com/r/88victory88/todoapp](https://hub.docker.com/r/88victory88/todoapp)
+
+## Building and Running the Application
+
+### Build the Docker Image
+```bash
+docker build -t todoapp:1.0.0 .
+
+Run the Docker Container
+bash
+Copy code

```

Comment:
The line 'bash' followed by 'Copy code' seems to be misplaced or unnecessary. If these are meant to be instructions, they should be clarified or removed if not needed.